cfgov/v1/management/commands/invalidate_single_page.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from wagtail.contrib.wagtailfrontendcache.utils import purge_url_from_cache
+
+
+class Command(BaseCommand):
+    help = 'Invalidates a page by its full URL'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--url',
+            nargs='?',
+            help='The full URL for the page you wish to invalidate'
+        )
+
+    def handle(self, *args, **options):
+        page_url = options['url']
+        if page_url:
```

Comment:
Addressed in https://github.com/cfpb/cfgov-refresh/pull/2858/commits/18d28b2615f06c60a68cc42bcf31b73dd60b0841. I kept the `--url` usage because then it's guaranteed to fail if someone doesn't know how to use it, as opposed to someone passing in some other parameter and it failing at the Akamai step.